kuma/core/decorators.py

```diff
@@ -136,3 +137,17 @@ def is_superuser(u):
 
 superuser_required = user_passes_test(is_superuser)
 #: A decorator to use for requiring a superuser
+
+
+def block_user_agents(view_func):
+    blockable_user_agents = []
+    if hasattr(settings, 'BLOCKABLE_USER_AGENTS'):
+        blockable_user_agents = settings.BLOCKABLE_USER_AGENTS
```

Comment:
Those three lines can be written as `blockable_user_agents = getattr(settings, 'BLOCKABLE_USER_AGENTS', [])`